SECURITY.md

```diff
@@ -0,0 +1,95 @@
+
+## Scope
+
+This policy applies to the following systems and services:
+- immudb database core application, 
+- immuadmin and immuclient applications
+- Golang Immudb SDK
+Source code of the above tools [is available on GH](https://github.com/codenotary/immudb).
+Also:
+- Java Immudb SDK ([repo link](https://github.com/codenotary/immudb4j))
+- Python Immudb SDK ([repo link](https://github.com/codenotary/immudb-py))
+- .Net Immudb SDK ([repo link](https://github.com/codenotary/immudb4dotnet))
+
+*To decide:*
+- Ruby on rails SDK ([repo link](https://github.com/ankane/immudb-ruby))
+- Nodejs SDK ([repo link](https://github.com/codenotary/immudb-node))
+
+
+Any service not expressly listed above, such as any connected services, are excluded from scope and are not authorized for testing. Additionally, vulnerabilities found in systems from our vendors fall outside of this policy’s scope and should be reported directly to the vendor according to their disclosure policy (if any). If you aren’t sure whether a system is in scope or not, contact us at (*enter email address here*) before starting your research (or at the security contact for the system’s domain name listed in the .gov WHOIS).
+
+Though we develop and maintain other internet-accessible systems or services, we ask that active research and testing only be conducted on the systems and services covered by the scope of this document. If there is a particular system not in scope that you think merits testing, please contact us to discuss it first. We will increase the scope of this policy over time.
+
+## Reporting a vulnerability
+
+We accept vulnerability reports via (*enter email address or other way*). Reports may be submitted anonymously. If you share contact information, we will acknowledge receipt of your report within 3 business days.
+
+We do not support PGP-encrypted emails.
+
+What we would like to see from you
+In order to help us triage and prioritize submissions, we recommend that your reports:
+
+- Describe the location the vulnerability was discovered and the potential impact of exploitation.
+- Offer a detailed description of the steps needed to reproduce the vulnerability (proof of concept scripts or screenshots are helpful).
+- Be in English, if possible.
+
+What you can expect from us
+- When you choose to share your contact information with us, we commit to coordinating with you as openly and as quickly as possible.
+- Within 3 business days, we will acknowledge that your report has been received.
+- To the best of our ability, we will confirm the existence of the vulnerability to you and be as transparent as possible about what steps we are taking during the remediation process, including on issues or challenges that may delay resolution.
+
+We will maintain an open dialogue to discuss issues.
+
+## Supported Versions
+
+Versions of immudb that are supported with security updates are listed in the table below.

```

Comment:
We should separately list immudb SDK versions here I believe

Reply:
I added it, but I doubt that this is the proper way. I think that each repo should have its own security / disclosure policy and maintain the version list separately.

Reply:
Yes, that sounds reasonable for non-go SDKs, but for this particular repository we should at least split between immudb / immuadmin / immuclient and go sdk